tutorials/deployment/slurm/ray-sbatch-job.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+
+# USAGE: sbatch --{optional-flags} ray-sbatch-job.sh
+# EXAMPLE: RUN_SCRIPT="python curation_script.py" sbatch --time=00:30:00 --nodes=2 ray-sbatch-job.sh
+
+########################################################
+# SLURM Directives
+########################################################
+#SBATCH --exclusive
+#SBATCH --output=curator-%j.out
+
+# NOTE: Some commonly used options are commented out below, uncomment them if you need to set them or add your own
+# #SBATCH --job-name=curator-pipeline
+# #SBATCH --nodes=1
+# #SBATCH --time=02:00:00
+# #SBATCH --account=my_account
+# #SBATCH --partition=my_partition
+# #SBATCH --dependency=singleton
+
+
+set -ux
+
+
+########################################################
+# User knobs (override via env vars) or in the script
+########################################################
+: "${RUN_COMMAND:=python -c 'import ray; ray.init(); print(ray.cluster_resources())'}" # Change as needed
+
+# Ports
+: "${GCS_PORT:=6379}"          # Ray GCS (native) port
+: "${CLIENT_PORT:=10001}"      # Ray Client port (ray://)
+: "${DASH_PORT:=8265}"         # Dashboard port
+
+
+########################################################
+# Container specific variables
+########################################################
+: "${IMAGE:=nvcr.io/nvidia/nemo-curator:25.09}"
```

Comment:
```suggestion
: "${IMAGE:=nvcr.io/nvidia/nemo-curator}"
```
Should work? We could also add a comment saying that this script is for 25.09 and above.